### PR TITLE
Use more logging when possible

### PR DESCRIPTION
**Pull request by lmaxwell24**
With integration with advantagescope, this allows for 
1. controller logging
2. replacing print statements with datalogmanager messages
3. logs the PDH
4. logs swerve expected positions vs actual

### CODE REVIEW

File: subsystems/loggingsubsystem.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        self.pdh = PowerDistribution(1, PowerDistribution.ModuleType.kRev)
+        self.oi = oi
+        self.dsTable = NetworkTableInstance.getDefault().getTable(
+            constants.kJoystickKeyLogPrefix
+        )
+
+    def periodic(self) -> None:
+        SmartDashboard.putData(self.pdh)
+
+        for controller in self.oi.controllers.values():
```
Comment (LandonBayer) [inline subsystems/loggingsubsystem.py:27]: What is the point of all this

**Review by LandonBayer — APPROVED** [review]
Code wise looks fine, only concern is increased lag/filled packets being an issue with a whole extra nettable set

**Comment by lmaxwell24** [comment]
Would not get flooded
NT4’s pub/sub model prevents unneeded packet flooding

File: subsystems/loggingsubsystem.py (hunk 1 above)
Comment (lmaxwell24) [reply subsystems/loggingsubsystem.py:27]: this is a pythonic implementation of controllers being logged to the driver station log, and consequentially I used the format that advantagekit uses for advantagescope for pushing joysticks

Comment (lmaxwell24) [reply subsystems/loggingsubsystem.py:27]: https://github.com/Mechanical-Advantage/AdvantageScope/blob/main/docs/tabs/JOYSTICKS.md

**Review by agelsomini — APPROVED** [review]

**Pull request merged**